Tests/MusicXMLTests/ScoreTests/ScoreTestSuite.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  ScoreTestSuite.swift
+//  MusicXMLTests
+//
+//  Created by Ben Lu on 10/14/19.
+//
+
+import XCTest
+import MusicXML
+
+class ScoreTests: XCTestCase {
+
+    func testAll() throws {
+        let paths = try FileManager.default.contentsOfDirectory(atPath: testSuiteURL.path)
+        for path in paths {
+            let testCaseURL = testSuiteURL.appendingPathComponent(path)
+            guard directoryExists(at: testSuiteURL.path) else {
+                continue
+            }
+            let manifestURL = testSuiteURL.appendingPathComponent(path).appendingPathComponent("manifest.plist")
```

Comment:
I haven't posted this anywhere (one day we will get there), but I'd like to keep lines to a 100-column width. Can we do something like this:

```Swift
let manifestURL.testSuiteURL
    .appendingPathComponent(path)
    .appendingPathComponent("manifest.plist")
```

Reply:
I will do. We may want to introduce linters to help us enforce the code standard. https://github.com/realm/SwiftLint